source/opt/passes.cpp

```diff
@@ -41,5 +41,27 @@ bool StripDebugInfoPass::Process(ir::Module* module) {
   return modified;
 }
 
+bool FreezeSpecConstantValuePass::Process(ir::Module* module) {
+  bool modified = false;
+  module->ForEachInst([&modified](ir::Instruction* inst) {
+    if (inst->opcode() == SpvOp::SpvOpSpecConstant) {
```

Comment:
I think this would read better as a switch statement.  And in that case you don't need the local variable for t inst->opcode()